source/core/src/main/com/csse3200/game/components/tasks/PauseTask.java

```diff
@@ -1,14 +1,18 @@
 package com.csse3200.game.components.tasks;
 
+import com.badlogic.gdx.utils.Logger;
 import com.csse3200.game.entities.Entity;
 import com.csse3200.game.components.ConfigComponent;
 import com.csse3200.game.entities.configs.*;
+import com.csse3200.game.services.ServiceLocator;
 
 /**
```

Comment:
Not specific to this PR, but might be good for overall modularity to have this task inherit from ProximityTask and override certain functions. Haven't looked into too much detail so not sure how viable this is though.

Reply:
Hi, it inherits from chase task which has a lot of the same metrics and calculations. ChaseTask inherits from this. Do you still want this overridden?